.github/workflows/ci-dgraph-net-e2e-tests.yml

```diff
@@ -0,0 +1,48 @@
+name: ci-dgraph-net-e2e-tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - master
+  schedule:
+    - cron: "0 0 * * *" # run workflow daily
+
+defaults:
+  run:
+    working-directory: source/Dgraph.tests.e2e
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup .NET Core SDK 7.0.x
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 7.0.x
+
+      - name: Spin up Dgraph
+        run: |
+          mkdir dgraph_data
+          docker pull dgraph/standalone:latest
+          docker run --name dgraph -d -p "8080:8080" -p "9080:9080" -v dgraph_data:/dgraph dgraph/standalone:latest

```

Comment:
do we need to do this mount of dgraph_data folder inside the container? Do we do this in dgo?

Reply:
Looks like the other clients to similar things inside their test scripts.

Reply:
dgo client doesn't seem to do it https://github.com/dgraph-io/dgo/blob/master/.github/workflows/ci-dgo-tests.yml. We use the docker compose file and we do not mount a data dir inside https://github.com/dgraph-io/dgo/blob/master/t/docker-compose.yml

Reply:
I think we can remove the -v flag from the command, it's not very useful for the tests here.

Reply:
Will it work properly if it's just `docker run --name dgraph -d -p "8080:8080" -p "9080:9080" dgraph/standalone:latest` ? Will it just pick some default data directory?

Reply:
I think so. It'll just not Mount the directory on the host. You should also remove the mkdir command before this.

Reply:
Done